commands/show/cluster/command.go

```diff
@@ -347,6 +347,25 @@ func printResult(cmd *cobra.Command, cmdLineArgs []string) {
 	clusterDetailsV4, clusterDetailsV5, nodePools, clusterStatus, credentialDetails, err := getClusterDetails(args)
 	if err != nil {
 		errors.HandleCommonErrors(err)
+
+		headline := ""
+		subtext := ""
+
+		switch {
+		case errors.IsClusterNotFoundError(err):
+			headline = "Cluster not found"
+			subtext = fmt.Sprintf("Either there is no cluster with ID '%s', or you have no access to it.\n", args.clusterID)
+			subtext += "Please check whether the cluster is listed when executing 'gsctl list clusters'."
```

Comment:
Why would `gsctl list clusters` help when cluster is not found?

Reply:
For two reasons

1. If the cluster the user was trying to display is shown, the entry can be used to double-check the spelling of the ID
2. for double-checking that the cluster is in fact not there (not accessible or not existing)